cli/src/main/java/org/jboss/as/cli/handlers/ifelse/ExpressionParser.java

```diff
@@ -138,6 +138,12 @@ protected Operand parseOperand() throws CommandLineException {
                     } else {
                         --depthCount;
                     }
+                } else if (input.charAt(endIndex) == '"') { // traverse over string operands
+                    endIndex = input.indexOf('"', endIndex + 1);
```

Comment:
I'll leave the approval to @aloubyansky or @Gonthim but if escaped quotes, `\"`, are allowed in strings then this won't work.